sdk/mobile-center-crashes/src/test/java/com/microsoft/azure/mobile/crashes/WrapperSdkExceptionManagerTest.java

```diff
@@ -39,29 +42,40 @@ public void constructWrapperSdkExceptionManager() {
     public void loadWrapperExceptionData() throws Exception {
         PowerMockito.doThrow(new IOException()).when(StorageHelper.InternalStorage.class);
         StorageHelper.InternalStorage.readObject(any(File.class));
-        WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID());
+        assertNull(WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID()));
         PowerMockito.doThrow(new ClassNotFoundException()).when(StorageHelper.InternalStorage.class);
         StorageHelper.InternalStorage.readObject(any(File.class));
-        WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID());
-
-        WrapperSdkExceptionManager.loadWrapperExceptionData(null);
-
+        assertNull(WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID()));
+        assertNull(WrapperSdkExceptionManager.loadWrapperExceptionData(null));
     }
 
     @Test
     public void deleteWrapperExceptionDataWithNullId() throws Exception {
         WrapperSdkExceptionManager.deleteWrapperExceptionData(null);
+        verifyStatic(Mockito.never());
+        StorageHelper.InternalStorage.delete(any(File.class));
+
+        WrapperSdkExceptionManager.deleteWrapperExceptionData(UUID.randomUUID());
+        verifyStatic(Mockito.never());
+        StorageHelper.InternalStorage.delete(any(File.class));
     }
 
     @Test
     public void saveWrapperExceptionData() throws IOException {
+        // cannot save with a null uuid
+        WrapperSdkExceptionManager.saveWrapperExceptionData(null, null);
+        verifyStatic(Mockito.never());
+        StorageHelper.InternalStorage.writeObject(any(File.class), anyString());
+
+        // ok to save null if there is a uuid
+        WrapperSdkExceptionManager.saveWrapperExceptionData(null, UUID.randomUUID()); //save null data
+        verifyStatic();
+        StorageHelper.InternalStorage.writeObject(any(File.class), any(Serializable.class));
```

Comment:
Need more than this. You can also check filename that starts with the given UUID.